scripts/set_runparams.py

```diff
@@ -30,7 +30,8 @@ def create_run_params(sim_dir, srf_name=None, workflow_config=None):
         workflow_config = load_config.load(
             os.path.dirname(os.path.realpath(__file__)), "workflow_config.json")
     global_root = workflow_config["global_root"]
-    tools_dir = workflow_config["bin_process_path"]
+    #tools_dir = workflow_config["bin_process_path"]
```

Comment:
Remove code if its no longer used? Not much point in keeping it around since its all in git anyways